compiler/ballerina-parser/src/test/resources/declarations/func-definition/func_params_source_27.bal

```diff
@@ -1,2 +1,2 @@
-function fn(*Configuration ...config, int k = 5) {
+function fn(*Configuration... config,intk=5) {
```

Comment:
We will have to skip only this test. This is due to issue #35240. Formatting `int k` here breaks the code.

Reply:
Let's add this test here (under the tests that should be enable in the future)

https://github.com/ballerina-platform/ballerina-lang/blob/3e65ae21fc773813d2d245b4576e8e69bacf03d5/misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java#L82